src/main/resources/db/changelog/alert/5.1.0/changelog-configuration-initialization.xml

```diff
@@ -16,4 +16,13 @@
             CALL DEFINE_FIELD('settings.saml.metadata.file', FALSE, 'component_settings','GLOBAL');
         </sql>
     </changeSet>
+    <!-- Register new user management fields. -->
+    <changeSet author="psantos" id="1567694372073-3">
+        <sql dbms="h2" stripComments="true">
+            CALL DEFINE_FIELD('settings.role.mapping.name.admin', FALSE, 'component_settings','GLOBAL');
```

Comment:
Did Eric add `DEFINE_DEFAULT_GLOBAL_FIELD` or something like that? It would remove two arguments per call.

Reply:
He created DEFINE_CHANNEL_FIELD. I will add an enhancement to create another procedure for global fields.